android/src/main/java/com/reactnativestripesdk/CardFormView.kt

```diff
@@ -134,6 +135,12 @@ class CardFormView(context: ThemedReactContext) : FrameLayout(context) {
       cardFormViewBinding.cardMultilineWidget.expiryDateEditText,
       cardFormViewBinding.postalCode
     )
+    val inputLayoutBindings = setOf(
+      cardFormViewBinding.cardMultilineWidget.cardNumberTextInputLayout,
+      cardFormViewBinding.cardMultilineWidget.cvcInputLayout,
+      cardFormViewBinding.cardMultilineWidget.expiryTextInputLayout,
+      cardFormViewBinding.postalCodeContainer,
+    )
```

Comment:
I think we can simplify the code in here just a little bit more:

let's use the same bindings for fontFamily that we do for placeholderColor. So here, let's change:
```suggestion
val placeholderTextBindings = setOf(
      multilineWidgetBinding.tlExpiry,
      multilineWidgetBinding.tlCardNumber,
      multilineWidgetBinding.tlCvc,
      cardFormViewBinding.postalCodeContainer,
    )
```

Then we can change the `placeholderColor?.let` block to:
```
placeholderColor?.let {
      for (binding in placeholderTextBindings) {
        binding.defaultHintTextColor = ColorStateList.valueOf(Color.parseColor(it))
      }
    }
```

and then I'll suggest the code changes below as well

Reply:
Changed.